pkgs/stdenv/generic/setup.sh

```diff
@@ -1200,35 +1233,7 @@ fixupPhase() {
 
 
     # Propagate dependencies & setup hook into the development output.
-    declare -ra flatVars=(
```

Comment:
Rest of the `fixupPhase` function does not seem to use any of these variables (`substituteAll` only captures environment variables) so this looks good to me.